payload-test.json

```diff
@@ -0,0 +1,146 @@
+{
+    "name": "AWS-Analytical-Dataset-Generation",
+    "releaseLabel": "emr-5.24.1",
+    "serviceRole": "analytical_dataset_generator",
+    "jobFlowRole": "analytical_dataset_generator",
+    "autoScalingRole": "arn:aws:iam::$ACC_NUMBER:role/AE_EMR_AutoScaling_Role",
```

Comment:
Is this a required field? We have no autoscaling requirement as this is an ephemeral cluster.

Reply:
yes it is.  according to CB